### PR TITLE
Enhance: Update dokan admin dashboard logo & add logo update notice.

### PR DESCRIPTION
**Pull request by MdAsifHossainNadim**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [x] I've included related pull request(s) (optional)
* [x] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Related Pull Request(s)

* Full PR Link
https://github.com/getdokan/dokan-pro/pull/3738

### Closes 

* Closes [#3321](https://github.com/getdokan/dokan-pro/issues/3321)

### How to test the changes in this Pull Request:

- Go to dokan admin dashboard & see menu logo, admin settings logo, admin logo notice etc.

### Changelog entry

```
Fix: Update dokan admin dashboard logo & add logo update notice.
```

Detailed Description of the pull request. What was previous behaviour
and what will be changed in this PR.

### Before Changes

image

### After Changes

Screenshot 2024-04-26 at 4 25 58 PM

### Feature Video (optional)

Link of detailed video if this PR is for a feature.

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

## Release Notes

- **New Features**
  - Updated admin notices with a new logo update notification.

- **Styling Improvements**
  - Enhanced styles for admin header, menu interactions, and various components to improve visual consistency and responsiveness.
  - Introduced new color variables for a unified color scheme across multiple components.
  - Updated button styles and background effects for better visibility and interaction.
  - Improved visual presentation of the vendor capabilities and premium pages for enhanced clarity.
  - Simplified background styles and enhanced layout for the Module Upgrade and Pro CTA components.
  - Enhanced media queries for better responsiveness of admin notices.

- **Bug Fixes**
  - Improved transition effects for admin notices, ensuring smoother animations and better visibility.

These updates aim to create a more cohesive and user-friendly interface for the admin panel.

### CODE REVIEW

**Comment by MdAsifHossainNadim** [comment]
@shohag121 Vhaiya, kindly review this PR at your convenience.

**Review by shohag121 — APPROVED** [review]

**Comment by shashwatahalder01** [comment]
Scenario:
The icon needs to update on several places on dokan

Screenshot at Apr 29 17-04-31
Screenshot at Apr 29 17-05-50
Screenshot at Apr 29 17-08-28
Screenshot at Apr 29 17-08-49

**Review by mrabbani — CHANGES_REQUESTED** [review]

**Comment by MdAsifHossainNadim** [comment]
@mrabbani Vhai, need to relevant ui's for [new issues](https://github.com/getdokan/dokan/pull/2258#issuecomment-2089508640).
@shashwatahalder01 Vhai, If anything is required or if you have any suggestions for us, then feel free to inform us.

**Comment by shashwatahalder01** [comment]
@MdAsifHossainNadim, @mrabbani,
It would be better if we could introduce a new color pallet that is aligned with the latest dokan logo

image

**Comment by tuhin480** [comment]
Here is the [Design Link](https://www.figma.com/file/XJkG32BCdIuu7PMYVJywZ7/Banner-Design?type=design&node-id=4533%3A6774&mode=design&t=vjpX03GC18EQm0Vn-1). Please have a look.

**Comment by MdAsifHossainNadim** [comment]
@tuhin480 Vhai, we need to make consistent between existing banner & updated logo. Here is the preview after updating logo.

Screenshot 2024-05-08 at 9 27 54 AM
Screenshot 2024-05-08 at 9 27 07 AM


I think It will be better if we added some shadow or spaces on left/bottom side in this banner image. (as like as previous)

image

Screenshot 2024-05-08 at 9 55 21 AM

We need to update wp org's banner & icon logo's as per latest update.

image (35)

**`CC:`** @mrabbani @ashrafux

**Comment by tuhin480** [comment]
This issue was split into two different GitHub links, which caused a delay. I've completed all the tasks and would like to request you to review the design.
[Design Link ](https://www.figma.com/design/8FadUe7m1y89LOoHAQffdG/Dokan-Plugin-Design?node-id=7753-1799&t=mITEGe6RpdTxBwXd-1)
Cc: @ashrafux @mrabbani

**Comment by mrabbani** [comment]
> This issue was split into two different GitHub links, which caused a delay. I've completed all the tasks and would like to request you to review the design. [Design Link ](https://www.figma.com/design/8FadUe7m1y89LOoHAQffdG/Dokan-Plugin-Design?node-id=7753-1799&t=mITEGe6RpdTxBwXd-1) Cc: @ashrafux @mrabbani @Mohaiminulislam1989

@imtiaz-pranto via, Pls review design and let us know your feedback.

**Comment by jamil-mahmud** [comment]
@shashwatahalder01 vaia, 

**Task:** 
- You will check the last worked branch of this issue and find each and every place in the plugin product in how many places Dokan New Logo is replaced. 
- You will update the issue list based on how many places new logo and branding are yet to be changed (Basic Logo, Icon, Branding Logo Banner)

**Resource:**
-- From our analysis, here is an impact list document where there are possibilities of changes. Still if you find extra places missed from the document kindly mention that in your report. --- [Impact List DOC](https://docs.google.com/document/d/1WEPp8deWB4DEvqGMfOdetFHZWUjcmg6Ed8J9zzNQEUA/edit?usp=sharing)

**Timeline**
- Reply me an estimated timeline when you will complete this. Thank You

**Comment by shashwatahalder01** [comment]
@jamil-mahmud bhai, I expect to complete this by tomorrow, October 22nd.

**Comment by shashwatahalder01** [comment]
@jamil-mahmud bhai, Here is the updated list where the dokan logo is not updated. The Analysis Team should also review and update the list if any scope has been missed.

- [ ] Dokan Premium Notice(1)
- [ ] Dokan Pro Features Page(3)
- [ ] Vendor Capabilities Page(2)
- [ ] Dokan Color Settings (1)
- [ ] Dokan Modules Page(1)

Screenshots:
Screenshot at Oct 22 09-42-49
Screenshot at Oct 22 09-46-04
Screenshot at Oct 22 09-47-52
Screenshot at Oct 22 09-48-37
Screenshot at Oct 22 09-52-12
Screenshot at Oct 22 09-56-26
Screenshot at Oct 22 10-27-52
Screenshot at Oct 22 10-50-31


N.B. The Dokan UI (Text color, Button background color, hyperlink, Dropdown menu, Banner) uses the previous Dokan icon theme. We should also consider that.

Screenshot at Oct 22 10-01-13
Screenshot at Oct 22 10-02-59
Screenshot at Oct 22 10-03-57
Screenshot at Oct 22 10-18-20
Screenshot at Oct 22 10-20-10

**Comment by jamil-mahmud** [comment]
@osmansufy vai
According to this [comment](https://github.com/getdokan/dokan/pull/2258#issuecomment-2428285351) all the required changes need to be done as per the [design](https://www.figma.com/design/8FadUe7m1y89LOoHAQffdG/Plugin-Product-Design?node-id=7241-8821&t=SxhCg5wOtlSDk7Ws-1)

**Task:**
- You will check the screenshots of the comments 
- The required changes are updated in the design
- About the color palette, it will be introduced as a secondary color for the product
   -- Conditions: (i) The color palette won't have any kind of effect on the existing users. (ii) New Color Pallet will only be a new option for the Admin users if they want to introduce newly
   
If anything needs to be required with the changes kindly let me know.

cc: @mrabbani vai, need assign allocation

File: src/admin/pages/Premium.vue [hunk 1]
```diff
@@ -1646,65 +1655,70 @@ export default {
...
-                }
-
-                svg {
-                    width: 15px;
-                    fill: #f2624d;
-                    margin-left: 6px;
-                }
-            }
-        }
+        //.cta-section {
```
Comment (MdAsifHossainNadim) [inline src/admin/pages/Premium.vue:1658]: Brother, kindly remove the unnecessary codes from the file.

File: assets/src/less/global-admin.less [hunk 2]
```diff
@@ -111,7 +111,7 @@
...
...2C145.155 75.331 135.487 62.1042 122.105 54.1373C113.842 49.1791 103.642 47.6922 97.5842 56.0645C96.6338 57.3779 95.8894 58.8748 95.1173 60.3003C88.6566 72.2286 57.2433 129.954 57.2093 130.017C57.2081 130.019 57.2087 130.018 57.2075 130.02C53.8062 136.284 54.5799 141.465 57.6734 145.719C62.3143 152.061 71.5189 154.923 79.0217 155.697C85.9832 156.47 93.022 155.697 99.7514 154.15C120.094 149.509 138.117 135.663 144.227 115.553C146.625 107.663 147.631 99.3866 146.78 91.1102Z' fill='white'/%3E%3Cdefs%3E%3ClinearGradient id='paint0_linear_1000_463' x1='100.945' y1='0.286865' x2='100.945' y2='200.287' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23FF7D92'/%3E%3Cstop offset='0.229167' stop-color='%23E3397D'/%3E%3Cstop offset='0.635417' stop-color='%23800CD2'/%3E%3Cstop offset='1' stop-color='%235812D0'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint1_linear_1000_463' x1='100.945' y1='0.286865' x2='100.945' y2='200.287' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23FF7D92'/%3E%3Cstop offset='0.229167' stop-color='%23E3397D'/%3E%3Cstop offset='0.635417' stop-color='%23800CD2'/%3E%3Cstop offset='1' stop-color='%235812D0'/%3E%3C/linearGradient%3E%3C/defs%3E%3C/svg%3E");
```
Comment (mrabbani) [inline assets/src/less/global-admin.less:114]: Pls  use `svg` file ref instead of code.

File: includes/Admin/Notices/Manager.php [hunk 3]
```diff
@@ -128,25 +128,25 @@ public function show_permalink_setting_notice( $notices ) {
...
-                'description' => __( 'Try it now to enhance your vendor\'s product upload experience, the older two-step version will be retired in one month.', 'dokan-lite' ),
-                'priority'    => 1,
+                'priority'          => 1,
                 'show_close_button' => true,
-                'ajax_data'   => [
-                    'action' => 'dismiss_dokan_one_step_product_notice',
-                    'nonce'  => wp_create_nonce( 'dismiss_dokan_one_step_product_nonce' ),
+                'type'              => 'info',
+                'title'             => __( 'New Dokan Logo!', 'dokan-lite' ),
+                'description'       => __( 'Introducing the new freshly designed Dokan Logo. Keep an eye on your admin dashboard!!', 'dokan-lite' ),
```
Comment (mrabbani) [inline includes/Admin/Notices/Manager.php:146]: @jamil-mahmud vai, Did you check the notice message of Admin panel.

File: src/admin/components/ProCta.vue [hunk 4]
```diff
@@ -75,26 +72,30 @@ export default {
     }
 
     .btn {
-        background-color: #fff;
-        color: #f2624d;
-        font-size: 16px;
+        background-color: #A244FF;
```
Comment (mrabbani) [inline src/admin/components/ProCta.vue:75]: Pls use variable for color code.

File: src/admin/components/ProCta.vue [hunk 5]
```diff
@@ -75,26 +72,30 @@ export default {
...
         font-weight: 300;
-        border: 1px solid #FF9381;
-        border-radius: 3px;
-        box-shadow: 0px 1px 2px 0px rgba(255, 128, 142, 0.05);
-        display: inline-block;
+        border-radius: 14px;
         text-decoration: none;
-        padding: 12px 27px;
+        display: inline-flex;
+        padding: 16.05px 45.167px;
```
Comment (mrabbani) [inline src/admin/components/ProCta.vue:83]: Pls remove the fractional pixel.

File: src/admin/components/ProCta.vue [hunk 6]
```diff
@@ -75,26 +72,30 @@ export default {
...
+        align-items: center;
 
         &:hover {
             box-shadow: 2.736px 7.518px 20px 0 rgba(0,0,0,.15);
```
Comment (mrabbani) [inline src/admin/components/ProCta.vue:87]: Pls remove the fractional pixel.

File: src/admin/components/ProCta.vue [hunk 7]
```diff
@@ -75,26 +72,30 @@ export default {
...
+            font-weight: 600;
+            line-height: normal;
+        }
         svg {
-            width: 15px;
-            fill: #f2624d;
-            margin-left: 6px;
+            width: 37.232px;
+            height: 30.243px;
         }
```
Comment (mrabbani) [inline src/admin/components/ProCta.vue:99]: pls remove the fractional pixel.

File: src/admin/pages/ChangeLog.vue [hunk 8]
```diff
@@ -365,7 +366,7 @@ ul {
                     background: none;
                     border-radius: 27px;
                     border: none;
-                    color: #cc7376;
+                    color: #5C626A;
```
Comment (mrabbani) [inline src/admin/pages/ChangeLog.vue:369]: pls use css variable instead.

File: src/admin/pages/VendorCapabilities.vue [hunk 9]
```diff
@@ -259,24 +255,36 @@ export default {
             }
         }
 
+
         .vendor-capabilities-banner {
             display: flex;
             align-items: center;
-            padding: 50px;
+            padding:50px;
```
Comment (mrabbani) [inline src/admin/pages/VendorCapabilities.vue:262]: Space is missing.

**Review by mrabbani — CHANGES_REQUESTED** [review]

File: assets/src/less/admin.less [hunk 10]
```diff
@@ -193,13 +195,13 @@
                             }
 
                             &:hover {
-                                color: #dd452c;
+                                color: @dokan-color;
 
                                 .dokan-icon {
-                                    background-color: rgba(221, 69, 44, 0.15);
+                                    background-color: rgba(239, 234, 255, 1);
```
[the remark this answers is not in the document]
Comment (osmansufy) [reply assets/src/less/admin.less:201]: yes we can , I  just tried to remain same pattern as before .

File: src/admin/components/ProCta.vue (hunk 7 above)
Comment (osmansufy) [reply src/admin/components/ProCta.vue:99]: @jamil-mahmud bhai I just follow the design , please tell the designer don't use fractional pixel.

File: src/admin/components/ProCta.vue (hunk 5 above)
Comment (osmansufy) [reply src/admin/components/ProCta.vue:84]: @jamil-mahmud bhai @tuhin480 bhai

Screenshot 2024-11-06 at 3 19 48 PM

File: src/admin/components/ProCta.vue (hunk 6 above)
Comment (osmansufy) [reply src/admin/components/ProCta.vue:88]: Bhai , can we use rem instead of pixel ?

**Comment by osmansufy** [comment]
Updated according to feedback

**Review by mrabbani — CHANGES_REQUESTED** [review]

File: includes/Admin/Notices/Manager.php (hunk 3 above)
Comment (jamil-mahmud) [reply includes/Admin/Notices/Manager.php:146]: @mrabbani vai
Title: Dokan came up with a new look!
Description: A new rebranded look is introduced in the entire platform. Check the updated visuals in different places. 

Recommending this change in the messages

File: assets/src/less/global-admin.less [hunk 11]
```diff
@@ -111,8 +111,8 @@
...
-                background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg viewBox='0 0 62 62' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='.90723' y='.79541' width='61' height='61' rx='30.5' fill='%23F16982'/%3E%3Cpath d='m19.688 25.014v-6.1614c1.4588-0.0303 7.7804 0.0301 13.407 3.6546-0.5844-0.2658-1.2264-0.4219-1.8846-0.4584-0.6581-0.0364-1.3177 0.0477-1.9361 0.2469-1.4936 0.5135-2.7441 1.8122-2.8483 3.2016-0.1737 2.3861 0 4.8627 0 7.2488v7.2186c-1.1115 0.0906-2.2577 0.1208-3.2649 0.1208-1.0768 0.0302-2.1188 0.0302-2.9524 0.1208l-0.521 0.0907v-15.283z' fill='%23fff'/%3E%3Cpath d='m17.848 43.77s-0.278-2.3257 2.5007-2.6579c2.7787-0.3323 8.0583 0.302 11.532-1.6008 0 0 2.0494-0.9363 2.4662-1.3893l-0.5558 1.6309s-1.6325 4.9534-6.5994 5.5876c-4.967 0.6041-5.9048-1.7517-9.3434-1.5705z' fill='%23fff'/%3E%3Cpath d='m28.546 45.824c3.9596-0.8457 8.4404-3.3828 8.4404-16.159 0-12.776-17.02-11.689-17.02-11.689 4.0639-2.084 25.008-4.6815 25.008 13.32 0 17.971-16.429 14.528-16.429 14.528z' fill='%23fff'/%3E%3C/svg%3E");
                 background-repeat: no-repeat;
+                background-image: url( "/assets/images/dokan-notice-logo.svg" );
```
[the remark this answers is not in the document]
Comment (osmansufy) [reply assets/src/less/global-admin.less:115]: @mrabbani bhai , svg already configure in our current webpack.config.js

I follow this .

still should I change the code ?
Screenshot 2024-11-07 at 10 53 45 AM

**Comment by osmansufy** [comment]
Realted PR on Dokan PRO

https://github.com/getdokan/dokan-pro/pull/3738

**Review by mrabbani — APPROVED** [review]

**Comment by shashwatahalder01** [comment]
Scenario:
 The logo isn't updated on the Dokan dashboard page



Screenshot:
Screenshot at Nov 12 14-26-30

**Comment by shashwatahalder01** [comment]
Scenario:
The logo isn't updated on the Dokan setup wizard

Screenshot:
Screenshot at Nov 12 14-32-38

**Comment by osmansufy** [comment]
@shashwatahalder01 & @jamil-mahmud  bhai ,

this is cash issue , try to check in incognito mode

**Comment by shashwatahalder01** [comment]
Scenario:
The logo isn't updated on the vendor profile

Screenshot:
Screenshot at Oct 22 10-58-15

**Comment by osmansufy** [comment]
> Scenario: The logo hasn't been updated on the Dokan shortcode page
> 
> Screenshot: Screenshot at Oct 22 11-02-50 Screenshot at Oct 22 11-03-11

@shashwatahalder01 Bhai . This will be changed from Dokan pro PR.

You can skip this in this Dokan Lite PR

**Comment by osmansufy** [comment]
> Scenario: The logo hasn't been updated on the Dokan color settings page
> 
> Screenshot: Screenshot at Nov 15 10-42-06

Updated bhai

**Comment by shashwatahalder01** [comment]
Scenario:
The image moves outside the viewport when opened.

Screenshot:
Screenshot at Nov 15 14-53-25
Screenshot at Nov 15 14-53-37

**Pull request merged**